docs/operator-notes.md

```diff
@@ -18,9 +18,18 @@ Any HTTP response code less than 500 results in the request being completed, and
 
 Ignition will initially wait 100 milliseconds between failed attempts, and the amount of time to wait doubles for each failed attempt until it reaches 5 seconds.
 
-## AWS and IAM roles
-
-Ignition has support for fetching files over the S3 protocol. When Ignition is running in Amazon EC2, it supports using the IAM role given to the EC2 instance to fetch protected assets from S3. If IAM credentials are not successfully fetched, Ignition will attempt to fetch the file with no credentials.
+## AWS S3 access
+
+Ignition has built-in support for fetching resources from the Amazon Simple Storage Service (AWS S3). Several URL formats are supported:
+
+| URL format | Semantics | Ignition behavior in Amazon EC2 instance | Ignition behavior outside EC2 |
+| - | - | - | - |
+| `s3://<bucket>/<object-path>` | Fetch the latest version of the object. | Authenticate using the instance's IAM role, if available. Fetch from the same AWS [partition](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html#arns-syntax) as the instance. | Fetch from the `aws` partition. |
+| `s3://<bucket>/<object-path>?versionId=<version>` | Fetch the specified version of the object. | Authenticate using the instance's IAM role, if available. Fetch from the same AWS partition as the instance. | Fetch from the `aws` partition. |
+| `arn:<partition>:s3:::<bucket>/<object-path>` | Fetch the latest version of the object from the specified partition. | Authenticate using the instance's IAM role, if available. | |
```

Comment:
It's not immediately obvious to me why the last column is left empty for this and the next row. Should we state why it's not applicable?

Reply:
Yeah, I was wondering about that too.  It's applicable, there's just nothing special to say about it.  Any ideas on how to represent that?